lib/pine.coffee

```diff
@@ -11,7 +11,7 @@ class PinejsClientRequest extends PinejsClientCore
 	# @summary Trigger a request to the resin.io API
 	# @private
 	#
-	# @description Makes use of [pinejs-client-js](https://bitbucket.org/rulemotion/pinejs-client-js)
+	# @description Makes use of [pinejs-client-js](https://github.com/resin-io/pinejs-client-js)
 	# You shouldn't make use of this method directly, but through models
 	#
 	# @param {Object} params - request params (same as node-request params)
```

Comment:
Not sure why you aren't just using the pinejs-client/request module here?

Anyway you need to change line 20 from `params.json = params.data` to `params.json = params.body`, since it will always be passed `body` now (I'd comment on the line directly but github won't let me :(  )